src/Service/DynamoDb/src/Enum/AttributeAction.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace AsyncAws\DynamoDb\Enum;
+
+class AttributeAction
+{
+    public const ADD = 'ADD';
+    public const DELETE = 'DELETE';
+    public const PUT = 'PUT';
+
+    public static function exists(string $value): bool
+    {
+        $values = [
```

Comment:
couldn't it be `static $values` for permformance?

Reply:
Im not sure. It leaving that for a future update. 
I think it is just minor improvements though..